test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1267,3 +1267,42 @@ UCS_TEST_P(test_ucp_am_nbx_rndv_memtype, rndv)
 }
 
 UCP_INSTANTIATE_TEST_CASE_GPU_AWARE(test_ucp_am_nbx_rndv_memtype);
+
+
+class test_ucp_am_nbx_rndv_memtype_disable_zcopy :
+        public test_ucp_am_nbx_rndv_memtype {
+protected:
+    void disable_rndv_zcopy_config(entity &e, uint64_t zcopy_caps)
+    {
+        ucp_ep_h ep             = e.ep();
+        ucp_ep_config_t *config = ucp_ep_config(ep);
+
+        if (zcopy_caps == UCT_IFACE_FLAG_PUT_ZCOPY) {
+            ucp_ep_config_rndv_zcopy_commit(0, &config->rndv.put_zcopy);
+        } else {
+            ASSERT_TRUE(zcopy_caps == UCT_IFACE_FLAG_GET_ZCOPY);
+            ucp_ep_config_rndv_zcopy_commit(0, &config->rndv.get_zcopy);
```

Comment:
what use case does it represent? We already run it on various transports, so what is emulated by hacking caps?

Reply:
we are emulating that either GET Zcopy or PUT Zcopy isn't supported by the transport.
the idea is to check how pipeline and normal RNDV will handle it.